src/makeInfinityAdder.js

```diff
@@ -3,8 +3,24 @@
 /***
  * @return {function}
  */
-function makeInfinityAdder() {
-  // write code here
+function makeInfinityAdder(initialValue = 0) {
```

Comment:
The `initialValue` parameter is not needed as per the task instructions. The `adder` function should start with a sum of 0 by default without providing an initial value.